app/controllers/static_pages_controller.rb

```diff
@@ -19,9 +19,9 @@ def mercury_update
     @page = StaticPage.friendly.find(params[:id])
     if @page.update_attributes(title: params[:content][:static_page_title][:value],
                                    body: params[:content][:static_page_body][:value])
-      render text: '' # So mercury knows it is successful
+      render html: ''
     else
-      render nothing: true
+      render html: ''
```

Comment:
@tansaku I wasn't able to  `render nothing: true` if I wanted mercury to work, since now is expecting `render html: ''`.

Maybe we can change the logic of the `if` statement to not render the same either way?